canopen/network.py

```diff
@@ -190,7 +190,7 @@ def send_periodic(self, can_id, data, period):
             An task object with a ``.stop()`` method to stop the transmission
         :rtype: canopen.network.PeriodicMessageTask
         """
-        return PeriodicMessageTask(can_id, data, period, self.bus)
+        return PeriodicMessageTask(can_id, data, period, self.bus, remote)
```

Comment:
The remote parameter will be passed as the extended parameter since it is positional. You should write `remote=remote`.

Reply:
Fixed, my bad. Lack of attention